### DISCUSSION TITLE
Compatibility with (again) new Thunderbird version (v115)

### DISCUSSION THREAD

**Discussion by droumanet** [Extended-Thunder/send-later#570] [General]
Hi,

I know this would be hard to restart to work on a plugin, just because Thunderbird team changed things (again).
But your plugin is so useful for users who always forgive attachment : just ask to send it one minute later, and I could remember I forget it !!!
Maybe you could ask for funding via Paypal (I've send $10 to XNote++ dev) : it's a small amount of money, but we're many to use your plugin ;)

Thanks,
David

**Comment by jikamens** [comment]
Money is not the problem, this is a duplicate of #551, and filing duplicate issues wastes time that developers would otherwise be spending working on porting Send Later to the new Thunderbird release.
